build_images.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+foreman_version=$1
```

Comment:
Just to be sure, please check if it's empty
```suggestion
foreman_version=$1
if [[ -z $foreman_version ]] ; then
  echo "Usage: $0 FOREMAN_VERSION"
  exit 1
fi
```